po/vi.po

```diff
@@ -19119,183 +19119,255 @@ msgid ""
 "comprehensive-rust/discussions). We would love to hear from you."
 msgstr ""
 
+# this is a translation, so the second part of the sentence need not be translated
 #: src/glossary.md
 msgid ""
 "The following is a glossary which aims to give a short definition of many "
 "Rust terms. For translations, this also serves to connect the term back to "
 "the English original."
 msgstr ""
+"Bảng từ vựng dưới đây cung cấp định nghĩa ngắn gọn của nhiều thuật ngữ Rust. "
 
 #: src/glossary.md
 msgid ""
 "allocate:  \n"
 "Dynamic memory allocation on [the heap](memory-management/stack-vs-heap.md)."
 msgstr ""
+"allocate:  \n"
+"Cấp phát bộ nhớ động trên [bộ nhớ heap](memory-management/stack-vs-heap.md)."
 
 #: src/glossary.md
 msgid ""
 "argument:  \n"
 "Information that is passed into a function or method."
 msgstr ""
+"argument:  \n"
+"Giá trị được truyền vào một hàm. Có thể được gọi là tham số."
 
 #: src/glossary.md
 msgid ""
 "Bare-metal Rust:  \n"
 "Low-level Rust development, often deployed to a system without an operating "
 "system. See [Bare-metal Rust](bare-metal.md)."
 msgstr ""
+"Bare-metal Rust:  \n"
+"Việc phát triển chương trình Rust ở để triển khai trên hệ thống không có hệ "
```

Comment:
Nit: `không có` -> `không cần`.

I think some contexts are still left untranslated such as "low-level", "deployed to a system". Can you translate them too?